<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectionClass</class>
 <widget class="QWidget" name="SelectionClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>757</width>
    <height>546</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SelectionClass</string>
  </property>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>12</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Language: </string>
   </property>
  </widget>
  <widget class="QPushButton" name="choiceButton">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>350</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Select file from computer</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>domain or ip of server :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>251</width>
     <height>211</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../../Spring 99.3/lms1/mainwindow.qrc">:/mainwindow/icon_ha/159110135480191692.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>300</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>View saved file  .hshvc</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>270</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Attend online meetings</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>350</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="ConnectButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>442</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <strikeout>false</strikeout>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>12</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>English</string>
    </property>
    <property name="icon">
     <iconset resource="../../Sumer 99.4/lmsAZ/mainwindow.qrc">
      <normaloff>:/mainwindow/icon_ha/en.PNG</normaloff>:/mainwindow/icon_ha/en.PNG</iconset>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Persian</string>
    </property>
    <property name="icon">
     <iconset resource="../../Sumer 99.4/lmsAZ/mainwindow.qrc">
      <normaloff>:/mainwindow/icon_ha/fa.PNG</normaloff>:/mainwindow/icon_ha/fa.PNG</iconset>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>50</y>
     <width>211</width>
     <height>241</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../../Spring 99.3/lms1/mainwindow.qrc">:/mainwindow/icon_ha/download.png</pixmap>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>400</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>save address of site</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>0</y>
     <width>81</width>
     <height>571</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../Spring 99.3/lms1/mainwindow.qrc"/>
  <include location="../../Sumer 99.4/lmsAZ/mainwindow.qrc"/>
 </resources>
 <connections/>
</ui>
